sale_order_currency_rate/models/sale_order.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Jarsa
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    currency_rate = fields.Float(
+        compute="_compute_currency_rate",
+        string="Currency",
```

Comment:
```suggestion
```

The currency string may lead to confusion in end users, if you don't define the string parameter it will take the field name, in this case "Currency Rate" that is acceptable